docs/development/build-android.md

```diff
@@ -16,84 +16,85 @@ The main non-standard dependency needed for building LiveCode is a Java Developm
 
 LiveCode requires both the Android Software Development Kit (SDK) and Native Development Kit (NDK).  You can download both from the [Android Developers site](https://developer.android.com/sdk/index.html).
 
-Extract both the NDK and SDK to a suitable directory, e.g. `~/android/toolchain`.  For example, you would run:
+Extract both the NDK and SDK to a suitable directory, e.g. `~/android/toolchain`.  For example, for the following values of `<host>`
+
+- mac: `darwin`
+- linux: `linux`
+- windows: `windows`
+
+you would run:
 
 ````bash
-mkdir -p ~/android/toolchain/android-sdk-linux
+mkdir -p ~/android/toolchain/android-sdk-<host>
 cd ~/android/toolchain
 
-wget https://dl.google.com/android/repository/android-ndk-r15-linux-x86_64.zip
-wget https://dl.google.com/android/repository/tools_r25.2.3-linux.zip
+wget https://dl.google.com/android/repository/android-ndk-r15-<host>-x86_64.zip
+wget https://dl.google.com/android/repository/tools_r25.2.3-<host>.zip
 
-unzip android-ndk-r15-linux-x86_64.zip
+unzip android-ndk-r15-<host>-x86_64.zip
 
-pushd android-sdk-linux
-unzip tools_r25.2.3-linux.zip
+pushd android-sdk-<host>
+unzip ../tools_r25.2.3-<host>.zip
 popd
 ````
 
 Update the SDK:
 
-    android-sdk-linux/tools/android update sdk --no-ui
+    android-sdk-<host>/tools/android update sdk --no-ui
 
-### Setting up the ARM toolchain
+(This command will download and install every Android SDK and will take some time).
+
+### Creating the toolchains
 
 Create a standalone toolchain (this simplifies setting up the build environment):
 
 ````bash
-android-ndk-r15/build/tools/make_standalone_toolchain.py \
-    --arch arm --api 16 --deprecated-headers \
-    --install-dir ${HOME}/android/toolchain/standalone
+for arch in arm arm64 x86 x86_64 ; do
```

Comment:
@livecodeali this did not work for me. For `arm64` and `x86_64` I had to use `--api 26`